Workout/Controller/WorkoutTVC.swift

```diff
@@ -56,7 +57,8 @@ class WorkoutTableViewController: UITableViewController, WorkoutDelegate {
 	func workoutLoaded(_ workout: Workout) {
 		DispatchQueue.main.async {
 			self.exportBtn.isEnabled = !self.workout.hasError
-			self.navigationItem.setRightBarButton(self.exportBtn, animated: true)
+            self.icsBtn.isEnabled = self.exportBtn.isEnabled
+			self.navigationItem.rightBarButtonItems = [self.exportBtn, self.icsBtn]
```

Comment:
Use the [explicit setter](https://developer.apple.com/documentation/uikit/uinavigationitem/1624939-setrightbarbuttonitems) to get a nicer animation when changing the buttons

Reply:
I have made the change.